pom.xml

```diff
@@ -248,6 +219,103 @@
     </build>
 
     <profiles>
+        <!--
+            When the `mp.specs.scope` property is not set the TS will 
+            build/execute the full set of MicroProfile specs modules.
+            E.g.: when testing Wildfly, which has all of the MicroProfile specs 
+            in since v. 19.0.0.Final, or when testing against EAP XP.
+        -->            
+        <profile>
+            <id>default-modules</id>
```

Comment:
Maybe something like "all-modules" would be better since we don't know from this if "default" equals "all". WDYT?

Reply:
Fixed